azavea/temperate
# Switch PlanItUser.email to a CIEmailField to make it case insensitive

## Overview

We've had multiple reports of users who try to login using a different uppercase/lowercase in their email than they did when signing up, and interpret their inability to login as a bug.

To accommodate this problem, we're switching email to be case insensitive.

## Testing Instructions

 * `scripts/update`
 * Ensure login is case insensitive
 * Ensure registration email uniqueness check is case insensitive
 * Ensure forgot password is case insensitive


Closes #1369 
Closes #1381 
Closes #1388
